shared/fs/sortbar.js

```diff
@@ -18,6 +23,57 @@ const stylesSortBar = {
   borderTopColor: globalColors.black_05,
   borderTopWidth: 1,
   minHeight: isMobile ? 24 : 24,
+  paddingLeft: '16px',
 }
 
-export default SortBar
+const SortBar = ({
+  sortSettingText,
```

Comment:
lets not destructure props that are used once. just keep it as props and use props.setSortByNameAsc etc